docs/04-operation-guides/security/sec-06-access-expose-grafana.md

```diff
@@ -4,6 +4,8 @@ title: Access and Expose Grafana and Jaeger
 
 > **CAUTION:** Jaeger is [deprecated](https://kyma-project.io/blog/jaeger-deprecation) and is planned to be removed. If you want to install a custom Jaeger stack, take a look at [this tutorial](https://github.com/kyma-project/examples/tree/main/jaeger).
 
+> **CAUTION:** Prometheus and Grafana are [deprecated](https://kyma-project.io/blog/2022/12/9/monitoring-deprecation) and are planned to be removed. If you want to install a custom Prometheus/Grafana stack, take a look at [this tutorial](https://github.com/kyma-project/examples/tree/main/prometheus).
```

Comment:
Why do you use "Caution" here, but "Note" in all the other docs? I assume it's because the Jaeger note above is a "Caution" type. 

One way or the other, we must be consistent and should pick one type in all places.
Follow our [content guidelines](https://kyma-project.io/community/guidelines/content-guidelines/03-formatting/#panels) to choose which type is a better match:

> The blue NOTE panel to point to something specific, usually relating to the topic.
> The red CAUTION panel to call attention to something critical that can cause inoperable behavior.

Also, it's usually not pretty to have two note boxes right next to each other. Consider merging both statements in one note box, like this:

```suggestion
> **CAUTION:** Kyma has deprecated the following components and plans to remove them. You can replace them with a custom stack:
  - To replace [deprecated](https://kyma-project.io/blog/jaeger-deprecation) Jaeger, take a look at [Install custom Jaeger in Kyma](https://github.com/kyma-project/examples/tree/main/jaeger).
  - To replace [deprecated](https://kyma-project.io/blog/2022/12/9/monitoring-deprecation) Prometheus and Grafana, take a look at [Install a custom kube-prometheus-stack in Kyma](https://github.com/kyma-project/examples/tree/main/prometheus).
```

Reply:
Unfortunately, I have seen that there is no consistency in the choice of "Caution" or "Note" for the deprecation comments. For example, "Note" is used [here](https://kyma-project.io/docs/kyma/main/05-technical-reference/00-architecture/obsv-02-architecture-logging/), while "Caution" is used [here](https://kyma-project.io/docs/kyma/main/05-technical-reference/00-architecture/obsv-03-architecture-tracing/). So, I just decided to go with the "Note" type, but I only used "Caution" in this page to be consistent with the Jaeger note above as you mentioned.

Reply:
I see. So if it's just for information, we should switch the ones that are "Caution" to "Note". Or if it can cause inoperable behaviour, we should switch the "Note" boxes to "Caution". 

Regardless which type you choose, I recommend combining these two boxes into one.

Reply:
The 2 boxes are now combined into one.
In addition, I changed the deprecation warnings for the Jaegar component from the "Caution" format to the "Note" format to be consistent with the deprecation warnings for Loki and Prometheus/Grafana